integration_test/house_mocked_integration2_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:house_stats/main.dart';
+import 'package:patrol/patrol.dart';
+
+main() {
+  patrolTest(
+    'house place widget shows red color for first place',
+    ($) async {
+      await $.pumpWidgetAndSettle(
+        const HouseStatsApp(),
+      );
+
+      expect($('1house'), findsOneWidget);
+      expect($('10'), findsOneWidget);
+      await $('1house').tap();
+
+      expect($('House details'), findsOneWidget);
+      expect($('1house'), findsOneWidget);
+    },
+    nativeAutomation: true,
```

Comment:
the order has to be this way, otherwise it won't compile. Perhaps you are using a different version of dart for your example projects?

![image](https://github.com/prolificcoder/house_stats/assets/149176/b3d2564a-e96a-4fd6-9719-62fd9320979a)

Reply:
That's because you're using Dart older than 2.17.

https://dart.dev/guides/language/evolution#dart-217

> Dart 2.17 removed some restrictions on named arguments. Named arguments can now be freely interleaved with positional arguments.